harpoon

```diff
@@ -47,12 +49,39 @@ view() {
     _switch "$name" "$path"
 }
 
-add() {
-    extended="$1"
+_getBookmark() {
+    extended=$1
+
     base_info='#{session_name}=#{session_path}'
     pane_info=':#{window_index}.#{pane_index}'
-    tmux display -p "$base_info$([ "$extended" = 1 ] && echo "$pane_info")" >>"$cachefile"
+    echo tmux display -p "$base_info$([ "$extended" = true ] && echo "$pane_info")"
```

Comment:
this echo is a bug, I'm sure this is going to cause an error, it will echo "tmux" and then complain that display is not a command. You can completely get rid of the leading echo here.